dlp/dlp_snippets/inspect_test.go

```diff
@@ -33,7 +33,7 @@ func TestInspectString(t *testing.T) {
 	}
 	for _, test := range tests {
 		buf := new(bytes.Buffer)
-		inspectString(buf, client, projectID, dlppb.Likelihood_POSSIBLE, 0, true, []string{"US_SOCIAL_SECURITY_NUMBER"}, test.s)
+		inspectString(buf, client, projectID, dlppb.Likelihood_POSSIBLE, 0, true, []string{"US_SOCIAL_SECURITY_NUMBER"}, []string{}, []string{}, test.s)
```

Comment:
Can do nil, which acts like an empty slice most of the time (for things like `range` and `len`).

Reply:
Done.